intercom-java/src/test/java/io/intercom/api/UserTest.java

```diff
@@ -246,6 +246,13 @@ public void TestSerdes() throws Exception {
         assertEquals(null, customAttribute);
     }
 
+    @Test
+    public void TestSerdesNoAvatar() throws Exception {
```

Comment:
Is this a possible typo?

Reply:
More of based on `TestSerdes` https://github.com/intercom/intercom-java/blob/7416b9b8ee1c06b01268a4055f57431e92ac53b7/intercom-java/src/test/java/io/intercom/api/UserTest.java#L171 of testing the [Serializer/Deserializer](https://en.wikipedia.org/wiki/SerDes) 

Think it would be clearer as just `TestNoAvatar`?

Reply:
Let's keep it as it is then so we don't change test names in this smaller PR. But I'd probably change the test name at a future stage to be easy to read and know what it tests. 👍